provision_scripts/install_intro_notebooks.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# This script clones the intro course to the VM
+set -x
+# Get the intro notebooks from GitHub.
+cd ~
+# Install git and zip/unzip just in case.
+sudo apt-get install -qq git
+sudo apt-get install -qq zip unzip
```

Comment:
`unzip` is already installed in `provision_scripts/install_common.sh` at the beginning of provisioning. I think it is cleaner to remove it here. If you require `zip`, it might also be cleaner to move installation to `provision_scripts/install_common.sh`...?